src/services/notifications/notification.hpp

```diff
@@ -124,6 +129,12 @@ class Notification
 	/// explicitly closed by the user.
 	Q_INVOKABLE void dismiss();
 
+	/// Send an inline reply to the notification with an inline reply action.
+	/// > [!WARNING] This method will only run if the notification
+	/// > has an action with identifier "inline-reply"
```

Comment:
Since this is user facing, phrase it in terms of exposed properties, e.g. "This method can only be called if @@hasInlineReply is true."